src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitChoiceGroupTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System.Drawing;
+using Bit.Client.Web.BlazorUI.Components.ChoiceGroup;
+using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Inputs
+{
+    [TestClass]
+    public class BitChoiceGroupTests : BunitTestContext
+    {
+        [DataTestMethod,
+           DataRow(Visual.Fluent, true, true),
+           DataRow(Visual.Fluent, true, false),
+           DataRow(Visual.Fluent, false, true),
+           DataRow(Visual.Fluent, false, false),
+
+           DataRow(Visual.Cupertino, true, true),
+           DataRow(Visual.Cupertino, true, false),
+           DataRow(Visual.Cupertino, false, true),
+           DataRow(Visual.Cupertino, false, false),
+
+           DataRow(Visual.Material, true, true),
+           DataRow(Visual.Material, true, false),
+           DataRow(Visual.Material, false, true),
+           DataRow(Visual.Material, false, false)
+       ]
+        public void BitChoiceGroupShouldRespectIsEnabled(Visual visual, bool groupIsEnabled, bool optionIsEnabled)
+        {
+            var component = RenderComponent<BitChoiceGroupTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.Visual, visual);
+                    parameters.Add(p => p.IsEnabled, groupIsEnabled);
+                    parameters.Add(p => p.Options, new()
+                    {
+                        new BitChoiceGroupOption
+                        {
+                            Key = "key1",
+                            IsEnabled = optionIsEnabled
+                        }
+                    });
+                });
+
+            var bitChoiceGroupOptions = component.Find(".bit-chgo");
+
+            var visualClass = visual == Visual.Cupertino ? "cupertino" : visual == Visual.Material ? "material" : "fluent";
+
+            if (groupIsEnabled is false || optionIsEnabled is false)
+            {
+                Assert.IsTrue(bitChoiceGroupOptions.ClassList.Contains($"bit-chgo-disabled-{visualClass}"));
+            }
```

Comment:
I think it is better to check that it doesn't have class `disabled` when the above condition is false.
```csharp
else 
{
  Assert.IsFalse(bitChoiceGroupOptions.ClassList.Contains($"bit-chgo-disabled-{visualClass}"));
}
```